exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = true; // finish this statement
```

Comment:
Hi Dharma, I learnt from others that here we were not expected to hardcode the answer. So it's meant to 
moreStudentsThanMentors  = studentCount  > mentorCount ;

The same thing is required for the rest of the other problems: roomMaxCapacity and sameName.